### PR TITLE
Driver improvements

### PR DESCRIPTION
**Pull request by antroseco**

### CODE REVIEW

File: tests/misc/process_args.c3 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @require_once "std/arithmetic.c3"
-@require_once "std/syscalls.c3"
+@require_once "syscalls.c3"
```
Comment (antroseco) [inline tests/misc/process_args.c3:2]: Including it as `"syscalls.c3"` is fine for `std` library code, not so great for user code.

File: driver.py [hunk 2]
```diff
@@ -1,76 +1,90 @@
...
     binary_output = Path("a.out")
 
+    load_target_config(args.target)
+
     parent_dir = Path(__file__).parent.resolve()
 
     if not args.nostdlib:
-        include_path.append(parent_dir)
+        args.include_path.append(parent_dir)
+        args.include_path.append(parent_dir / "std" / get_target())
```
Comment (DaveDuck321) [inline driver.py:87]: Cool

File: driver.py [hunk 3]
```diff
@@ -1,76 +1,90 @@
...
+    emit_optimized_llvm: bool = False
+    """Output IR optimized by clang to a file."""
+
+    emit_everything: bool = False
+    """Output generated IR, optimized IR, and the binary."""
+
+    debug_compiler: bool = False
+    """Print full exception traces."""
 
-    assert level != "2"  # Coward assert
```
Comment (DaveDuck321) [inline driver.py:33]: Pls fix, necessary feature

File: glang [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python3
+from driver import main
+
+main()
```
Comment (DaveDuck321) [inline glang:4]: What @FChippendale wanted

File: std/x86_64_linux/syscalls.c3 [hunk 5]
```diff
@@ -1,5 +1,5 @@
-@require_once "util.c3"
-@require_once "sys/uio.c3"
+@require_once "../util.c3"
+@require_once "../sys/uio.c3"
```
Comment (DaveDuck321) [inline std/x86_64_linux/syscalls.c3:2]: Suppose thats a feature now :smile:

File: target.py [hunk 6]
```diff
@@ -206,6 +206,13 @@ def _get_config() -> TargetConfig:
     return _target_config
 
 
+def get_target() -> str:
+    global _target
+
```
Comment (DaveDuck321) [inline target.py:211]: This is redundant right?

**Review by DaveDuck321 — CHANGES_REQUESTED** [review]
LGTM (but missing the coward assert)

File: driver.py (hunk 3 above)
Comment (antroseco) [reply driver.py:33]: Done...

File: target.py (hunk 6 above)
Comment (antroseco) [reply target.py:211]: wontfix

**Pull request merged**